### PR TITLE
[DONE] Translation of tools-model-mgmt.md ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/community/tools-model-mgmt.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+* **[Baobab](https://github.com/Yomguithereal/baobab):** شجرة بيانات ثابتة تدعم المؤشرات.
+* **[cortex](https://github.com/mquan/cortex/):** مكتبة JavaScript لإدارة البيانات مركزيًا باستخدام React.
+* **[DeLorean](https://github.com/deloreanjs/delorean):** إطار جافا سكريبت محايد تمامًا لتطبيق مفاهيم Flux على واجهاتك بسهولة.
+* **[Elsa](https://github.com/JonAbrams/elsa):** إضافة Babel التي تجعل المصفوفات والكائنات الخاصة بك غير قابل للتغيير بشكل افتراضي ... ويمنحهم قوى عظمى!
+* **[Immutable](https://github.com/facebook/immutable-js):** هياكل البيانات غير القابلة للتغيير المصممة لتزيد من الكفاءة والبساط.
+* **[js-data](https://www.js-data.io/):** إطار محايد لتخزن بيانات للواجهة الأمامية ، من شأنه أيضًا جلب بياناتك.
+* **[McFly](https://github.com/kenwheeler/mcfly):** مكتبة Flux خفيفة، تضيف مصانع للتخزين والإجراءات.
+* **[NuclearJS](https://github.com/optimizely/nuclear-js):** بنية Flux تفاعلية غير قابل للتغيير. واجهة مستخدم محايدة.
+* **[Reflux](https://github.com/spoike/refluxjs):** تنفيذ قائم على الأحداث لبنية Flux متكامل مع مكونات React.تنفيذ قائم على الأحداث لبنية Flux متكامل مع مكونات React.
```
Comment (Fcmam5) [inline content/community/tools-model-mgmt.md]: Duplication :stuck_out_tongue: and `البنية` is `مؤنث` so we need the `ـة`
```suggestion
* **[Reflux](https://github.com/spoike/refluxjs):** تنفيذ قائم على الأحداث لبنية Flux متكاملة مع مكونات React.
```

File: content/community/tools-model-mgmt.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+* **[backbone-react-component](https://github.com/magalhas/backbone-react-component):** استخدم نموذج ومجموعات Backbone المتعددة مع مكونات React على جهتي العميل والخادم.
+* **[Baobab](https://github.com/Yomguithereal/baobab):** شجرة بيانات ثابتة تدعم المؤشرات.
+* **[cortex](https://github.com/mquan/cortex/):** مكتبة JavaScript لإدارة البيانات مركزيًا باستخدام React.
+* **[DeLorean](https://github.com/deloreanjs/delorean):** إطار جافا سكريبت محايد تمامًا لتطبيق مفاهيم Flux على واجهاتك بسهولة.
+* **[Elsa](https://github.com/JonAbrams/elsa):** إضافة Babel التي تجعل المصفوفات والكائنات الخاصة بك غير قابل للتغيير بشكل افتراضي ... ويمنحهم قوى عظمى!
+* **[Immutable](https://github.com/facebook/immutable-js):** هياكل البيانات غير القابلة للتغيير المصممة لتزيد من الكفاءة والبساط.
+* **[js-data](https://www.js-data.io/):** إطار محايد لتخزن بيانات للواجهة الأمامية ، من شأنه أيضًا جلب بياناتك.
+* **[McFly](https://github.com/kenwheeler/mcfly):** مكتبة Flux خفيفة، تضيف مصانع للتخزين والإجراءات.
+* **[NuclearJS](https://github.com/optimizely/nuclear-js):** بنية Flux تفاعلية غير قابل للتغيير. واجهة مستخدم محايدة.
```
Comment (Fcmam5) [inline content/community/tools-model-mgmt.md]:
```suggestion
* **[NuclearJS](https://github.com/optimizely/nuclear-js):** بنية Flux تفاعلية غير قابلة للتغيير. واجهة مستخدم محايدة.
```

File: content/community/tools-model-mgmt.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+* **[Alt](https://alt.js.org/):** مكتبة لـ Flux نقية و أصلية صنعت بطريقة متماثلة و بدون شيفرة متكررة.
+* **[astarisx](https://zuudo.github.io/astarisx/):** إطار MVVM القابل للتكوين العالي لـ React مع جهاز توجيه pushState مدمج.
+* **[avers](https://github.com/wereHamster/avers):** مكتبة تجريد حديثة لنموذج جهة العميل.
+* **[backbone-react-component](https://github.com/magalhas/backbone-react-component):** استخدم نموذج ومجموعات Backbone المتعددة مع مكونات React على جهتي العميل والخادم.
+* **[Baobab](https://github.com/Yomguithereal/baobab):** شجرة بيانات ثابتة تدعم المؤشرات.
+* **[cortex](https://github.com/mquan/cortex/):** مكتبة JavaScript لإدارة البيانات مركزيًا باستخدام React.
+* **[DeLorean](https://github.com/deloreanjs/delorean):** إطار جافا سكريبت محايد تمامًا لتطبيق مفاهيم Flux على واجهاتك بسهولة.
+* **[Elsa](https://github.com/JonAbrams/elsa):** إضافة Babel التي تجعل المصفوفات والكائنات الخاصة بك غير قابل للتغيير بشكل افتراضي ... ويمنحهم قوى عظمى!
+* **[Immutable](https://github.com/facebook/immutable-js):** هياكل البيانات غير القابلة للتغيير المصممة لتزيد من الكفاءة والبساط.
```
Comment (Fcmam5) [inline content/community/tools-model-mgmt.md]:
```suggestion
* **[Immutable](https://github.com/facebook/immutable-js):** هياكل البيانات غير القابلة للتغيير المصممة لتزيد من الكفاءة والبساطة.
```

File: content/community/tools-model-mgmt.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-* **[Reflux](https://github.com/spoike/refluxjs):** An event-based implementation of the Flux architecture that integrates with React components.
+* **[Alt](https://alt.js.org/):** مكتبة لـ Flux نقية و أصلية صنعت بطريقة متماثلة و بدون شيفرة متكررة.
+* **[astarisx](https://zuudo.github.io/astarisx/):** إطار MVVM القابل للتكوين العالي لـ React مع جهاز توجيه pushState مدمج.
+* **[avers](https://github.com/wereHamster/avers):** مكتبة تجريد حديثة لنموذج جهة العميل.
+* **[backbone-react-component](https://github.com/magalhas/backbone-react-component):** استخدم نموذج ومجموعات Backbone المتعددة مع مكونات React على جهتي العميل والخادم.
+* **[Baobab](https://github.com/Yomguithereal/baobab):** شجرة بيانات ثابتة تدعم المؤشرات.
+* **[cortex](https://github.com/mquan/cortex/):** مكتبة JavaScript لإدارة البيانات مركزيًا باستخدام React.
+* **[DeLorean](https://github.com/deloreanjs/delorean):** إطار جافا سكريبت محايد تمامًا لتطبيق مفاهيم Flux على واجهاتك بسهولة.
+* **[Elsa](https://github.com/JonAbrams/elsa):** إضافة Babel التي تجعل المصفوفات والكائنات الخاصة بك غير قابل للتغيير بشكل افتراضي ... ويمنحهم قوى عظمى!
```
Comment (Fcmam5) [inline content/community/tools-model-mgmt.md]:
```suggestion
* **[Elsa](https://github.com/JonAbrams/elsa):** إضافة Babel التي تجعل المصفوفات والكائنات الخاصة بك غير قابلة للتغيير بشكل افتراضي ... ويمنحهم قوى عظمى!
```

File: content/community/tools-model-mgmt.md [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
-* **[cortex](https://github.com/mquan/cortex/):** A JavaScript library for centrally managing data with React.
-* **[DeLorean](https://github.com/deloreanjs/delorean):** A completely agnostic JavaScript framework to apply Flux concepts into your interfaces easily.
-* **[Elsa](https://github.com/JonAbrams/elsa):** A Babel plugin that makes your arrays and objects immutable by default… and gives them super powers!
-* **[Immutable](https://github.com/facebook/immutable-js):** Immutable data structures designed to be more JavaScript-y than Mori.
-* **[js-data](https://www.js-data.io/):** A framework-agnostic frontend datastore, that will also fetch your data.
-* **[McFly](https://github.com/kenwheeler/mcfly):** A lightweight Flux library that adds factories for Stores & Actions.
-* **[NuclearJS](https://github.com/optimizely/nuclear-js):** Immutable, reactive Flux architecture. UI Agnostic.
-* **[Reflux](https://github.com/spoike/refluxjs):** An event-based implementation of the Flux architecture that integrates with React components.
+* **[Alt](https://alt.js.org/):** مكتبة لـ Flux نقية و أصلية صنعت بطريقة متماثلة و بدون شيفرة متكررة.
```
Comment (Fcmam5) [inline content/community/tools-model-mgmt.md]: I'm not sure about making `شيفرة مكتررة` as a translation for `boilerplate`.
A boilerplate text in Arabic is `نص نموذجي`, which is the text that we take an example to start from and then we can continue writing or we can change in it...
The first time I read `شيفرة مكتررة` I thought that the original text was `repetitive code`  or `reused code (put in some context)`, my suggestions:

```suggestion
* **[Alt](https://alt.js.org/):** مكتبة لـ Flux نقية و أصلية صنعت بطريقة متماثلة و بدون المشروع النموذجي (boilerplate).
```

or
```suggestion
* **[Alt](https://alt.js.org/):** مكتبة لـ Flux نقية و أصلية صنعت بطريقة متماثلة و بدون الشيفرة النموذجية (boilerplate).
```

File: content/community/tools-model-mgmt.md (hunk 1 above)
Comment (3imed-jaberi) [reply content/community/tools-model-mgmt.md]: it's typo 🤣

**Comment by 3imed-jaberi** [comment]
> As you say `بَاهي` 😄 ❤️

 `بزاف` ❤️ 🇩🇿

**Pull request merged**